UI/data/themes/Yami.qss

```diff
@@ -82,7 +82,7 @@ QWidget {
     selection-background-color: rgb(40,76,184);
     selection-color: palette(text);
     font-size: 10pt;
-    font-family: 'Open Sans', Helvetica, Arial, 'MS Shell Dlg', sans-serif
+    font-family: 'Open Sans', '.AppleSystemUIFont', Helvetica, Arial, 'MS Shell Dlg', sans-serif;
```

Comment:
Should not this applied to every Yami variants (Grey, Light, Acri, Rachni) ?

Reply:
Correct, fixed.